Source/WebCore/html/shadow/DataListButtonElement.h

```diff
@@ -49,6 +49,7 @@ class DataListButtonElement final : public HTMLDivElement {
     explicit DataListButtonElement(Document&, DataListButtonOwner&);
 
     void defaultEventHandler(Event&) override;
+    bool isDisabledFormControl() const override { return shadowHost() && shadowHost()->isDisabledFormControl(); }
```

Comment:
`isDisabledFormControl` is a virtual function so the static analyzer can't determine it to be safe.